lib/ssl/test/ssl_test_lib.erl

```diff
@@ -2902,24 +2960,34 @@ openssl_tls_version_support(Proto, Opts, Port, Exe, Args0) ->
 init_protocol_version(Version, Config)
   when Version == 'dtlsv1.2'; Version == 'dtlsv1' ->
     ssl:stop(),
-    application:load(ssl),
-    application:set_env(ssl, dtls_protocol_version, [Version]),
     ssl:start(),
-    NewConfig = proplists:delete(protocol_opts, proplists:delete(protocol, Config)),
-    [{protocol, dtls}, {protocol_opts, [{protocol, dtls}]} | NewConfig];
+    Group0 = proplists:get_value(group_opts, Config, []),
+    Group = proplists:delete(version, proplists:delete(protocol, Group0)),
```

Comment:
here and below.
```
*:delete(version` or `*:delete(versions
                                      ^
```
?

version for group,
but versions for Config?

Reply:
Yeah this is how it always have been in the tests. 
You set a list of versions in options, but in the testcase we test a specific version, so I guess it is ok.

Reply:
you later delete version from Config, also ok?